TikokulympicBeta/Models/API/ArrivalRequest.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ArrivalRequest.swift
+//  TikokulympicBeta
+//
+//  Created by 株丹優一郎 on 2024/09/17.
+//
+
+import Foundation
+import Alamofire
+
+struct ArrivalResponse: ResponseProtocol {
+    let message: String
+}
+
+struct ArrivalRequest: RequestProtocol {
+    typealias Response = ArrivalResponse
+    var method: HTTPMethod {.post}
+    var path: String {"/attendances/\(eventId)/\(userId)"}
+    var parameters: Parameters? {
```

Comment:
デフォルトでnilが入ってこっちのparametersの変更が潰されてnilになるかも
```suggestion
    var parameters: Parameters? {
```